tests/integration/test_api/test_config/test_DOS_blocking_system/test_DOS_blocking_system.py

```diff
@@ -0,0 +1,170 @@
+"""
+copyright: Copyright (C) 2015-2023, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: These tests will check if the 'DOS' (Denial-of-service attack) blocking feature of the API handled
+       by the 'wazuh-apid' daemon is working properly. The Wazuh API is an open source 'RESTful' API
+       that allows for interaction with the Wazuh manager from a web browser, command line tool
+       like 'cURL' or any script or program that can make web requests.
+
+components:
+    - api
+
+suite: config
+
+targets:
+    - manager
+
+daemons:
+    - wazuh-apid
+    - wazuh-modulesd
+    - wazuh-analysisd
+    - wazuh-execd
+    - wazuh-db
+    - wazuh-remoted
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/api/getting-started.html
+    - https://documentation.wazuh.com/current/user-manual/api/configuration.html#access
+    - https://en.wikipedia.org/wiki/Denial-of-service_attack
+
+tags:
+    - api
+"""
+import os
+import time
+import pytest
+import requests
+
+from wazuh_testing.constants.api import CONFIGURATION_TYPES, AGENTS_ROUTE
+from wazuh_testing.constants.daemons import API_DAEMONS_REQUIREMENTS
+from wazuh_testing.modules.api.helpers import get_base_url, login
+from wazuh_testing.utils.configuration import get_test_cases_data, load_configuration_template
+
+
+# Marks
+pytestmark = pytest.mark.server
+
+# Variables
+# Used by add_configuration to select the target configuration file
+configuration_type = CONFIGURATION_TYPES[0]
+
+# Paths
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configuration_folder_path = os.path.join(test_data_path, 'configuration_template')
+cases_folder_path = os.path.join(test_data_path, 'test_cases')
+test_configuration_path = os.path.join(configuration_folder_path, 'configuration_DOS_blocking_system.yaml')
+test_cases_path = os.path.join(cases_folder_path, 'cases_DOS_blocking_system.yaml')
+
+# Configurations
+test_configuration, test_metadata, test_cases_ids = get_test_cases_data(test_cases_path)
+test_configuration = load_configuration_template(test_configuration_path, test_configuration, test_metadata)
+daemons_handler_configuration = {'daemons': API_DAEMONS_REQUIREMENTS}
+
+
+# Tests
+@pytest.mark.tier(level=0)
```

Comment:
Don't you think it is more organized to put this in `pytestmark`? If so, apply this in all tests.

Reply:
I think this could be applied when all of the test functions share the same level. I applied this to 1 test module: `test_statistics_format.py` in 40a5f11